reporter/html.js

```diff
@@ -697,6 +697,14 @@ QUnit.testDone(function( details ) {
 
 	if ( bad === 0 ) {
 		addClass( assertList, "qunit-collapsed" );
+	} else {
+		if ( config.collapse === false ) {
+			config.collapse = -1;
```

Comment:
We shouldn't change the configuration from the HTML reporter. If you need to keep track of state, use local state (variables) within the reporter.